src/_ert_forward_model_runner/runner.py

```diff
@@ -23,14 +28,42 @@ def __init__(self, jobs_data):
 
         self._set_environment()
 
+    def _manifest_dict(self):
+        with open("manifest.json", mode="r", encoding="utf-8") as f:
+            data = json.load(f)
+        timeout = 10
```

Comment:
I'd say that the total timeout should be more than 10 seconds.

Reply:
Removed this logic after a discussion with @eivindjahren